pkg/install/generate.go

```diff
@@ -189,6 +189,37 @@ func InstallerPodSpec(
 		// 	Name:  "OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE",
 		// 	Value: "jdiaz-rhcos-4.3",
 		// })
+	case cd.Spec.Platform.VSphere != nil:
+		env = append(
+			env,
+			corev1.EnvVar{
+				Name: "GOVC_HOST",
```

Comment:
How does this environment variable operate in conjunction with the `.spec.platform.vsphere.vCenter` field in the ClusterDeployment? Does the creds secret really need this data?

Reply:
Only needed in the secret if we use the secret `GOVC_HOST` as a parameter to deprovision. I could use either.